### PR TITLE
Introduce "launch_status" config in the admin page instead of "deactivate" config. #4

### PR DESCRIPTION
**Pull request by gimite**
This "launch_status" can be either "prelaunch", "launched" or "deactivated".
With this config, we don't need to configure "active" and "launched" status separately.

### CODE REVIEW

File: app/admin.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 import const
 import reveal
 import tasks
+import logging
```
Comment (skywhale) [inline app/admin.py]: sort alphabetically

File: app/admin.py [hunk 2]
```diff
@@ -37,8 +38,9 @@ def get(self):
         user = users.get_current_user()
         simplejson.encoder.FLOAT_REPR = str
         encoder = simplejson.encoder.JSONEncoder(ensure_ascii=False)
-        config_json = dict((name, encoder.encode(self.config[name]))
-                           for name in self.config.keys())
+        view_config = self.__model_config_to_view_config(self.config)
+        view_config_json = dict((name, encoder.encode(value))
+                           for name, value in view_config.iteritems())
```
Comment (skywhale) [inline app/admin.py]: nit: align indentation

File: app/model.py [hunk 3]
```diff
@@ -124,8 +122,9 @@ def list_active(cls):
     @classmethod
     def list_launched(cls):
         """Returns a list of the launched (listed in menu) repository names."""
-        return [name for name in config.get('launched_repos', [])
-                if not config.get_for_repo(name, 'deactivated')]
+        return [name for name in Repo.list()
+                if config.get_for_repo(name, 'launched') and
+                   not config.get_for_repo(name, 'deactivated')]
```
Comment (skywhale) [inline app/model.py]: nit: one more space indentation

File: app/resources/admin.html.template [hunk 4]
```diff
@@ -231,67 +231,183 @@
...
-          these deletions do not generate expiry notices
-          in the outgoing PFIF feeds,
-          so any clients that import test records into their own repositories
-          cannot be expected to delete them on the same short schedule.
-	</div>
+        <div class="note">
+          <div>
+            <strong>Use case:</strong> This mode is usually used for always-on
+            repository such as "japan" repository, while no crisis is
+            on-going, so it is only used for testing purpose.
```
Comment (skywhale) [inline app/resources/admin.html.template]: maybe drop ", so it is only used for testing purpose". had a slight difficulty in parsing.

**Comment by skywhale** [comment]
LGTM

Thanks for the changeset :)

File: app/admin.py (hunk 1 above)
Comment (gimite) [inline app/admin.py]: It was not used. Removed.

File: app/admin.py (hunk 2 above)
Comment (gimite) [inline app/admin.py]: Done

File: app/model.py (hunk 3 above)
Comment (gimite) [inline app/model.py]: Done

File: app/resources/admin.html.template (hunk 4 above)
Comment (gimite) [inline app/resources/admin.html.template]: Done

**Comment by gimite** [comment]
Thanks for the review. Merged to the master branch.

**Pull request merged**